deploy/vuln-monitor/scan.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright 2021 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script scans the vulnerability report that is generated by Container Analysis.
+
+set -xeo pipefail
+# Variables that will be substituted by trigger configuration or valued provided through command line with --substitutions flag.
+if [ -z "$_TAG_FILTER" ]; then
+  _TAG_FILTER="v.*-lts|^edge$"
+fi
+if [ -z "$_BASE_IMAGE" ] ; then
+  _BASE_IMAGE="us-east1-docker.pkg.dev/k8s-skaffold/scanning/skaffold"
+fi
+# If changed, also change the same variable in report.sh.
+VULN_FILE=/workspace/skaffold_vuln.txt
+
+append() {
+  printf "%s\n" $1 >>$2
+}
+
+check_vulnerability(){
+  base_image=$1
+  result_file=$2
+  tags=$3
+  tags_filter=""
+
+  if [ -z "$tags" ]; then
+    # We should only scan lts images within 1 year window from the first patch of the release.
+    targeted_base_tags="$(gcloud container images list-tags "$base_image" --filter="timestamp.datetime > -P1Y AND tags~v.*\.1-lts" --format='value(tags)')"
+    for line in $targeted_base_tags; do
+      IFS=',' read -ra t <<< "${line}"
```

Comment:
nit: the other IFS call uses double quotes, might make sense to use consistent style